tests/src/Behat/CorporateSiteInformationContext.php

```diff
@@ -69,6 +69,45 @@ public function assertSiteOwner(string $label): void {
     }
   }
 
+  /**
+   * Set content owner configuration.
+   *
+   * @param string $label
+   *   Content owner SKOS concept label.
+   *
+   * @Given I set the content owner to :label
+   */
+  public function setContentOwner(string $label): void {
```

Comment:
setSiteDefaultContentOwner() otherwise it would make me think you set the field default value without the site in it.